src/Countries/Switzerland.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Switzerland extends Country
+{
+    public function countryCode(): string
+    {
+        return 'ch';
+    }
+
+    /** @return array<string, string|CarbonImmutable> */
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'Neujahr' => '01-01',
+            'Berchtoldstag' => '01-02',
+            'Bundesfeier' => '08-01',
+            'Weihnachtstag' => '12-25',
+            'Stephanstag' => '12-26',
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function variableHolidays(int $year): array
+    {
+        $easter = CarbonImmutable::createFromTimestamp(easter_date($year))
```

Comment:
Use `$this->easter($year)` instead. Only available if you rebased on this repo's main branch

Reply:
done